src/extensions/mvs/tree/molstar/molstar-tree.ts

```diff
@@ -29,6 +29,13 @@ export const MolstarTreeSchema = TreeSchema({
                 format: RequiredField(MolstarParseFormatT),
             },
         },
+        raw_volume: {
+            ...FullMVSTreeSchema.nodes.raw_volume,
+            params: {
+                source: RequiredField(RawVolumeSourceT),
+                options: OptionalField(RawVolumeOptionsT)
+            }
+        },
```

Comment:
If this node is exactly the same as in `MVSTreeSchema`, do not include it here explicitely. It is already included via `...FullMVSTreeSchema.nodes` on line 18.

The nodes explicitely listed here are only those that have different params in `MolstarTreeSchema` than in `MVSTreeSchema`.

Reply:
Right, thanks for the explanation, did not know that. Will remove it from here.